config/locales/pt-BR.yml

```diff
@@ -0,0 +1,175 @@
+pt-BR:
+  title: "RubyGems.org"
+  subtitle: "o host de gems da sua comunidade"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Accesso Negado. Por favor se registre em: https://rubygems.org"
+  feed_subscribed: "RubyGems.org | Gems do seu Feed"
+  feed_latest: "RubyGems.org | Últimas Gems"
+  time_ago: "%{duration} atrás"
+  form_disable_with: "Atualizando..."
+  update: "Atualizar"
+  edit: "Editar"
+  sign_in: "Logar"
+  sign_up: "Cadastrar"
+  footer_about: "RubyGems.org é o serviço de hospedagem da comunidade Ruby. Publique e instale suas gems instantaneamente. Use a API para interagir e encontrar mais informações sobre gems disponíveis. Torne-se um contribuidor e melhore o site com suas mudanças."
```

Comment:
Instead of `serviço de hospedagem da comunidade Ruby`, should it be `serviço de hospedagem de gems da comunidade Ruby`?